config/routes.rb

```diff
@@ -1,4 +1,8 @@
 Rails.application.routes.draw do
+  namespace :api do
+    get 'building_operator/create'
```

Comment:
we don't want to expose this api, so can probs remove this. Building operators can only be created on delegation, which you handled in your code `BuildingOperator.where(email: email).first_or_create`. By adding a route for building operator creation, we essentially allow anyone who makes a json post request to create a new building operator which is kinda dangerous.

Reply:
Yah my b I forgot to remove this